src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1805,6 +1805,160 @@
 
 	<xamarin:Style TargetType="GridViewItem"
 				   BasedOn="{StaticResource GridViewItemExpanded}" />
+	
+	<!--Default style for CalendarDatePicker -->
+	<Style TargetType="CalendarDatePicker">
+		<ios:Setter Property="FlyoutPlacement"
+					Value="Full" />
+		<Setter Property="Orientation"
+				Value="Horizontal" />
+		<Setter Property="IsTabStop"
+				Value="False" />
+		<Setter Property="FontSize"
+				Value="17" />
+		<!--	<Setter Property="Foreground"
+				Value="Red" /> -->
+		<Setter Property="Background"
+				Value="Transparent" />
+		<Setter Property="HorizontalAlignment"
+				Value="Center" />
+		<Setter Property="VerticalAlignment"
+				Value="Center" />
+		<Setter Property="Padding"
+				Value="0" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="CalendarDatePicker">
+					<StackPanel x:Name="LayoutRoot">
+						<VisualStateManager.VisualStateGroups>
+							<VisualStateGroup x:Name="CommonStates">
+								<VisualState x:Name="Normal" />
+								<VisualState x:Name="Disabled">
+									<Storyboard>
+										<ObjectAnimationUsingKeyFrames Storyboard.TargetName="HeaderContentPresenter"
+																	   Storyboard.TargetProperty="Foreground">
+											<DiscreteObjectKeyFrame KeyTime="0"
+																	Value="Black" />
+										</ObjectAnimationUsingKeyFrames>
+									</Storyboard>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+						<ContentPresenter x:Name="HeaderContentPresenter"
+										  x:DeferLoadStrategy="Lazy"
+										  Visibility="Collapsed"
+										  Content="{TemplateBinding Header}"
+										  ContentTemplate="{TemplateBinding HeaderTemplate}"
+										  Margin="0,0,0,8"
+										  Background="{TemplateBinding Background}"
+										  Foreground="{StaticResource Color02Brush}"
+										  AutomationProperties.AccessibilityView="Raw" />
+						<Button x:Name="FlyoutButton"
+								Style="{StaticResource DatePickerFlyoutButtonStyle}"
+								Foreground="{TemplateBinding Foreground}"
+								Background="{TemplateBinding Background}"
+								BorderThickness="{TemplateBinding BorderThickness}"
+								BorderBrush="{TemplateBinding BorderBrush}"
+								IsEnabled="{TemplateBinding IsEnabled}"
+								MinHeight="{TemplateBinding MinHeight}"
+								Margin="0"
+								Padding="0"
+								HorizontalAlignment="Stretch"
+								HorizontalContentAlignment="Stretch"
+								VerticalAlignment="Stretch"
+								VerticalContentAlignment="Stretch">
+							<StackPanel Orientation="Horizontal"
+										HorizontalAlignment="Center">
+								<TextBlock Text="{Binding _DateString, RelativeSource={RelativeSource TemplatedParent}}" />
```

Comment:
This does not seem correct. Does the original template from windows do this ? Binding to a private field is generally not a good idea.

Reply:
Probably it can be done using Converter, but I never wrote any (and I don't know if Converters are implemented in Uno).